bankingapp/apps/netbankingapp/models.py

```diff
@@ -0,0 +1,93 @@
+from django.db import models
+from django.core.validators import (
+    MinValueValidator,
+    MaxValueValidator,
+)
+import uuid
+
+# Create your models here.
+class Users(models.Model):
+
+    id = models.UUIDField(
+        primary_key=True, unique=True, editable=False,
+        default=uuid.uuid4
+        )
+
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30)
+
+    username = models.CharField(
+        max_length=30, unique=True, null=True, blank=True,
+        help_text=(
+            'Required. 30 characters or fewer. Letters, digits and '
+            'Use special characters in case the given username is repeated'
+        ))
+
+    dob = models.DateField(blank=False, null=False)
+    address = models.CharField(max_length=100)
+
+    password = models.CharField(
+        max_length=300, unique=False, null=True, blank=True,
+        help_text=(
+              'Required. 30 characters or fewer. Letters, digits and '
+              'special characters'
+          ))
+
+    def __str__(self):
+        return str(self.username)
+
+class Account(models.Model):
+    account_id = models.UUIDField(
```

Comment:
Inconsistent naming and indentation (compare id for user and account_id for Account